src/pages/landing/ebooks/components/_get-ebook.js

```diff
@@ -284,10 +290,12 @@ const GetEbook = ({ onSubmit, ebook_utm_code }) => {
 
     return submit_status === 'success' ? (
         <ResponseWrapper>
+            <div className="email-image-wrapper">
+                <EmailImage src={ViewEmailImage} alt="Email image" />
```

Comment:
I don't think we need the word `image` in alt tags. They usually handle that for us.